python/run-tests.py

```diff
@@ -281,6 +262,12 @@ def process_queue(task_queue):
     total_duration = time.time() - start_time
     LOGGER.info("Tests passed in %i seconds", total_duration)
 
+    for key, lines in sorted(SKIPPED_TESTS.items()):
+        pyspark_python, test_name = key
+        LOGGER.info("\nSkipped tests in %s with %s:" % (test_name, pyspark_python))
```

Comment:
Will it be too verbose to print all skipped tests? An option is to record them into LOG_FILE only. No strong preference here.

Reply:
Yea, I get it but I feel sure people felt it should be printed out in the console from previous discussions. I actually don't feel strongly too.

Anyway we could have only few of them eventually because more of them are by missing Pandas and Arrow so probably this could be fine.

No strong preference here too. let me know if other guys have a preference.